src/components/Editor/Overlay.tsx

```diff
@@ -91,6 +91,11 @@ function Overlay(): JSX.Element {
     addActionToList,
   } = useActionList(experienceData);
 
+  // useEffect for autoSave
+  useEffect(()=>{
+    experienceData.experience.actionList = [...actionList];
+    saveExp(userId, experienceData.experience);
+  }, [actionList, experienceData, userId])
```

Comment:
is it possible to set a timer for this so it doesn't save every time actionlist is updated?